### DISCUSSION TITLE
Does Empyrial support Chinese A-Share market?

### DISCUSSION THREAD

**Discussion by qj4chen** [ssantoshp/Empyrial#57] [Q&A] [answered]
I tested with several stock codes such as '600519.SH' but failed to fetch data

**Comment by santoshlite** [comment, marked as answer]
Hey!

I see....

Basically, it supports every symbol that is on Yahoo Finance (https://finance.yahoo.com/). 

So, you could use the search bar to see if it exists. However, it seems that yahoo finance doesn't recognize this ticker.

I checked and found this: https://finance.yahoo.com/quote/600519.SS/

If this is the symbol that you were looking for, try with "600519.SS" on Empyrial instead!

Hope it helps!

**Comment by qj4chen** [comment]
that's great! many thanks
